### PR TITLE
Move internal users to S3

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
From security point of view, hiding the internal_users.yml file and only retrieving it during deployment

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: nightly-playground/resources/security-config/internal_users.yml [hunk 1]
```diff
@@ -8,31 +8,19 @@ _meta:
 # Define your internal users here
```
Comment (prudhvigodithi) [inline nightly-playground/resources/security-config/internal_users.yml:8]: We can completely remove this file right @gaiksaya ?

File: .github/workflows/nightly-playground-deploy.yml [hunk 2]
```diff
@@ -77,6 +77,7 @@ jobs:
           npm install
           playground_id=`echo ${{inputs.dist_version}} | cut -d. -f1`x
           echo "PLAYGROUND_ID=$playground_id" >> "$GITHUB_OUTPUT"
+          aws s3 cp s3://nightly-playgrounds-snapshots-bucket/internal_users.yml resources/security-config/internal_users.yml
```
Comment (prudhvigodithi) [inline .github/workflows/nightly-playground-deploy.yml:80]: I assume this bucket is already pre-created and has the `internal_users.yml` stored, does this file gets updated for every new version ? or we maintain one standard file for all version?

File: nightly-playground/resources/security-config/internal_users.yml (hunk 1 above)
Comment (gaiksaya) [reply nightly-playground/resources/security-config/internal_users.yml:8]: tests are failing if I remove it. Also wanted to keep a sample for users to view/

**Review by prudhvigodithi — APPROVED** [review]

File: .github/workflows/nightly-playground-deploy.yml (hunk 2 above)
Comment (gaiksaya) [reply .github/workflows/nightly-playground-deploy.yml:80]: One standard for all versions. Yes its pre-created via https://github.com/opensearch-project/opensearch-devops/blob/main/nightly-playground/lib/common-tools-stack.ts#L45

**Pull request merged**